.github/workflows/ci.yml

```diff
@@ -303,8 +303,9 @@ jobs:
 
   tas_tests_search_api:
     name: ${{ matrix.testSuite }} | TAS tests (Search API)
-    runs-on: ubuntu-latest
+    runs-on: alfrescoPub-ubuntu2204-16G-4CPU
     if: >
+      (github.ref_name == 'master' || startsWith(github.ref_name, 'release/') || github.event_name == 'pull_request') &&
```

Comment:
I'm not sure this is useful given the `on.pull_request.branches` &  `on.push.branches` configs.